package.json

```diff
@@ -7,5 +7,10 @@
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
dunno if im completely wrong but are you meant to commit the pack.json and package-lock.json files?

Reply:
not sure! I haven't changed the .gitignore at all, so I would assume if they're committing they're supposed to... I'll check